src/renderer/windows/GPMWebView/index.js

```diff
@@ -37,6 +37,21 @@ const waitForExternal = setInterval(() => {
     require('gmusic-mini-player.js')(GMusic);
     const GMusicTheme = require('gmusic-theme.js');
 
+    // Replace requestAnimationFrame for invoking what we want, even when window is minimized
+    const defaultAnimationFrame = window.requestAnimationFrame;
+    window.requestAnimationFrame = (f) => {
+      if (f.toString) {
+        const str = f.toString();
```

Comment:
This is going to dramatically slow down animations, stringifying functions is a computation heavy task.